koku/api/report/aws/query_handler.py

```diff
@@ -83,8 +83,11 @@ def annotations(self):
             annotations['usage_units'] = Coalesce(self._mapper.usage_units_key, Value(units_fallback))
         # { query_param: database_field_name }
         fields = self._mapper.provider_map.get('annotations')
+        prefix_removed_parameters_list = map(lambda x: x if ':' not in x else x.split(':', maxsplit=1)[1],
+                                             self.parameters.get('group_by', {}).keys())
+        parameters_list = self.parameters.get('group_by', {}).keys()
```

Comment:
You don't need the parameters_list now that you created the prefix_removed_parameters_list

Reply:
@adberglund i thought so too, until the unit tests failed, and then i put them both there and now the unit tests pass

Reply:
@adberglund fixed.